packages/common/src/chains/kovan.json

```
@@ -62,13 +62,13 @@
     },
     {
       "name": "berlin",
-      "block": null,
-      "forkHash": null
+      "block": 24770900,
+      "forkHash": "0x1a0f10d9"
     }, 
     {
       "name": "london",
-      "block": null, 
-      "forkHash": null
+      "block": 26741100,
+      "forkHash": "0x1ed20b71"
```

I am not seeing this value in the PR you linked, I see `"eip1559Transition": "0x198096c"`
```suggestion
      "forkHash": "0x198096c"
```

---

I did the local check with `calkForkHash()` and the values Andrew provided indeed seem correct. Maybe the introductory text to the PR was just a bit misleading.

Will merge.

---

> I am not seeing this value in the PR you linked, I see `"eip1559Transition": "0x198096c"`
> ```suggestion
>       "forkHash": "0x198096c"
> ```
> 

So, I thought those hex strings in the OpenEthereum PRs were the fork hashes but when I ran our tests, the ones related to forkhashes failed. So, I just used our `calcForkHash` method to get them and inserted. I'm not sure what those strings from the OE PRs represent.